src/Jupyter/Jupyter.csproj

```diff
@@ -8,39 +8,40 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Remove="res\kernel.js" />
-    <None Remove="res\logo-64x64.png" />
-  </ItemGroup>
+  <PropertyGroup>
+    <NoWarn>0162</NoWarn>
+    <Authors>Microsoft</Authors>
+    <Description>Microsoft's IQ# Library with Jupyter Extensions.</Description>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageReleaseNotes>See: https://docs.microsoft.com/en-us/quantum/relnotes/</PackageReleaseNotes>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/Microsoft/IQSharp</PackageProjectUrl>
+    <PackageIconUrl>https://secure.gravatar.com/avatar/bd1f02955b2853ba0a3b1cdc2434e8ec.png</PackageIconUrl>
+    <PackageTags>Quantum Q# Qsharp</PackageTags>
+    <PackageId>Microsoft.Quantum.IQSharp.Jupyter</PackageId>
+    <ContentTargetFolders>\</ContentTargetFolders>
+    <ApplicationIcon />
+  </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\build\DelaySign.cs" Link="Properties\DelaySign.cs" />
+    <Compile Remove="res\**" />
+    <EmbeddedResource Remove="res\**" />
+    <None Remove="res\**" />
+    <TypeScriptCompile Remove="res\**" />
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="res\logo-64x64.png" />
+    <Compile Include="..\..\build\DelaySign.cs" Link="Properties\DelaySign.cs" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Jupyter.Core" Version="1.3.52077" />
-    <PackageReference Include="System.Reactive" Version="4.3.2" />
+    <PackageReference Include="Microsoft.Jupyter.Core" Version="1.3.9294-CI-20200417-004906" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Core\Core.csproj" />
   </ItemGroup>
 
   <!-- TypeScript compilation -->
-  <ItemGroup>
```

Comment:
This is a good change, it was always a bit weird that the TypeScript had to be built from a different project than the kernel itself.

Reply:
Now that you point it out, I should also remove the comment saying that the TypeScript compilation is here.